### ISSUE TITLE
`DateTimeCell` type attribute is not set when writing XLSX

### ISSUE THREAD

**Issue by benhurrell** [openspout/openspout#109]
When using `openspout/openspout` v4.8.0, submitting a `DateTimeCell` to the XLSX writer does not add the 'type' attribute to the generated XML. Instead the default fallback of a numeric type is used.

I've created a branch at https://github.com/benhurrell/openspout/tree/fix-cell-date-attribute that fixes this. I began writing a test to 'prove' the fix works, but have the following questions:
- None of the other type attribute mappings appear to be tested; therefore is a test for this fix necessary/helpful?
- If a test is required, are there any helper classes in the test suite I should look to use?

Thank you for your time.

**Comment by Slamdunk** [comment]
Yes tests are necessary; I don't know if current test suite is suited for your regression test though

**Comment by underdpt** [comment]
Hi,

I've just been hit by this.

I see that test are neccesary, I would like to contribute but don't know where to start. Maybe having a first test for attribute mapping could serve as a guidance.

**Comment by Slamdunk** [comment]
You may look at https://github.com/openspout/openspout/blob/e4179a2b68ccd35fad52e60b0c7ebbf8ad9fe677/tests/Writer/XLSX/WriterTest.php#L298 for a good starting point
